spec/test_games.rb

```diff
@@ -0,0 +1,43 @@
+def test_game_one
```

Comment:
Have you covered enough games? I think there's more permutations in the final frame that could be tested. It's hard to completely assert that everything works as intended from looking at the unit tests.